app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisitedViewHolder.kt

```diff
@@ -64,9 +64,11 @@ class RecentlyVisitedViewHolder(
             onRecentVisitClick = { recentlyVisitedItem, pageNumber ->
                 when (recentlyVisitedItem) {
                     is RecentHistoryHighlight -> {
+                        metrics.track(Event.HistoryHighlightOpened)
```

Comment:
Can we move this into the controller method that is called  by `onRecentHistoryHighlightClicked` and add a test to verify this call?

Reply:
This can be a followup.